site2/tools/apidoc/python/pydoctor-generator.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
I know that these two script files duplicate each other. But since scripts are less combinable, and this is a specific small scope, I'd like to keep it simple stupid :)